pkg/ephemeral-disk-utils/utils.go

```diff
@@ -33,6 +33,36 @@ import (
 	"kubevirt.io/client-go/log"
 )
 
+// TODO this should be part of structs, instead of a global
+var DefaultOwnershipManager OwnershipManagerInterface = &OwnershipManager{uid: 107, gid: 107}
```

Comment:
hrm... well on systems I'm familiar with I recall UID 107 is qemu, but I don't think we can guarantee this is universal. recommend using `user.LookupId` if possible.

Reply:
addressed